### PR TITLE
Adding a path for persistent data

### PR DESCRIPTION
**Pull request by vaage**
To support persistent data with the server, this change adds a command line argument that specifies where cross run data can be stored.

### CODE REVIEW

File: run_server.sh [hunk 1]
```diff
@@ -17,10 +17,11 @@
 TEAM_ID="$1"
 TEAM_SECRET="$2"
 PORT="$3"
-RELAY_ADDRESS="$4"
+PERSISTENT="$4"
```
Comment (crepricg) [inline run_server.sh]: the name "Persistent" is not clear.

File: run_server.sh [hunk 2]
```diff
@@ -17,10 +17,11 @@
...
+RELAY_ADDRESS="$5"
 
-if [[ "TEAM_ID" == "" || "$TEAM_SECRET" == "" || "$PORT" == "" ]] ; then
-  echo 'usage: <TEAM ID> <TEAM SECRET> <PORT> [RELAY ADDRESS]'
+if [[ "TEAM_ID" == "" || "$TEAM_SECRET" == "" || "$PORT" == "" || "$PERSISTENT" == "" ]] ; then
+  echo 'usage: <TEAM ID> <TEAM SECRET> <PORT> <PERSISENT> [RELAY ADDRESS]'
```
Comment (crepricg) [inline run_server.sh]: we may want to print something more here. The command line starts getting complicated.

Comment (vaage) [inline run_server.sh]: What would you suggest?

File: run_server.sh (hunk 1 above)
Comment (vaage) [inline run_server.sh]: What would you suggest?

File: run_server.sh (hunk 2 above)
Comment (wgenem) [inline run_server.sh]: persistent is misspelled in line 24.
Either add a single line for each parameter that explains what the parameter is used for, or add a line that points to the README (and add the usage information to the README).

File: run_server.sh (hunk 1 above)
Comment (wgenem) [inline run_server.sh]: I agree.  Persistent what?  Is it a path or a flag or something else?

**Pull request merged**